lib/machinery_helper.rb

```diff
@@ -52,15 +57,19 @@ def inject_helper
   end
 
   def run_helper(scope)
-    json = @system.run_command(
-      File.join(
-        Machinery::REMOTE_HELPERS_PATH, "machinery-helper"
-      ), stdout: :capture, stderr: STDERR
-    )
+    json = @system.run_command(@remote_helpers_path, stdout: :capture, stderr: STDERR)
```

Comment:
The code would be a little bit clearer and robust, if it would not use the member variable directly, but a method to access the remote helper path.